phpBB/language/en/cli.php

```diff
@@ -110,8 +110,10 @@
 
 	'CLI_EXTENSION_DISABLE_FAILURE'		=> 'Could not disable extension %s',
 	'CLI_EXTENSION_DISABLE_SUCCESS'		=> 'Successfully disabled extension %s',
+	'CLI_EXTENSION_ALREADY_DISABLED'	=> 'Extension %s was already disabled',
```

Comment:
Should I change "Extension %s was already enabled" to "Extension %s is enabled" too?